src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -182,40 +177,17 @@ public function editAction(Post $post, Request $request)
      */
     public function deleteAction(Request $request, Post $post)
     {
-        $form = $this->createDeleteForm($post);
-        $form->handleRequest($request);
+        if (!$this->isCsrfTokenValid('delete', $request->request->get('token'))) {
+            return $this->redirectToRoute('admin_post_index');
```

Comment:
The request could come from "edit page", is right to redirect to index page? shouldn't be `admin_post_edit`?

In the other hand, could you add a flash messages (error) when this happen? Similar to:
```
The CSRF token is invalid. Please try to resubmit the form.
```